src/Stratis.CirrusPegD/Program.cs

```diff
@@ -99,8 +102,11 @@ private static IFullNode GetSidechainFullNode(string[] args)
                 MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
             };
 
+            NetworkType networkType = nodeSettings.Network.NetworkType;
+
             var fedPegOptions = new FederatedPegOptions(
-                counterChainNetwork: MainChainNetworks[nodeSettings.Network.NetworkType]()
+                counterChainNetwork: MainChainNetworks[networkType](),
+                walletSyncFromHeight: new int[] { 1, 1, 1 }[(int)networkType] // TODO
```

Comment:
Think `1, 1, 1` makes sense on this side, we can remove the TODO.

Or maybe these should stay as 1 and we get the node owners to run with an optional extra param to speed things up 🤔

Reply:
No harm in going with that line of thought for now.